### DISCUSSION TITLE
File path separation commands only work once

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [R-nvim/R.nvim#191] [General]
Not quite sure how but the file path separation commands (`sp` and `sh`) only work when I first open the file and run it and not after that. Is there anything I'm missing about how to use them? 
Running them after the first time does absolutely nothing.

**Comment by PMassicotte** [comment]
It works here, is your cursor on the path?

![Peek 2024-07-19 07-25](https://github.com/user-attachments/assets/4ab51530-1913-47ac-8017-54f9f29f8c90)

**Reply by lokesh-krishna** [reply]
It is. Like I said, it works the first time but not after that. It doesn't throw any error either. Simply does nothing. Are there any logs I can find?

**Comment by PMassicotte** [comment]
What do you mean "not after that".

**Comment by jalvesaq** [comment]
You can check if there are map conflicts with the command `:nmap <LocalLeader>s`

**Comment by lokesh-krishna** [comment]
![image](https://github.com/user-attachments/assets/32c70ef3-e1c0-4a23-aa3c-3d1df63a4529)
Doesn't look like it.  
How can I check what exactly it's doing when I use the keybinding? It doesn't even emit any errors or warnings.

**Comment by PMassicotte** [comment]
@lokesh-krishna The function has been updated. Please check section 6.38.

**Comment by lokesh-krishna** [comment]
So cool! Set it to `here` but still have the same issue. Works once and then doesn't. Would really like to know how to troubleshoot.

**Comment by PMassicotte** [comment]
Try to make it replicable. I tested many different scenarios and all work for me.
